### ISSUE TITLE
Wo Long

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#175]

**Comment by aflies1** [comment]
Resume; https://drive.google.com/drive/folders/1nWwkVJCtvCkvgcx3xfonrgqEFwSh67dO

**Comment by ldaudet** [comment]
Wo Long

He's a math guy adding some CS in his skillset. Sort of a funny guy, connected with a background with small aliens in a spaceship working behind him.

He has a lot of experience in machine learning, which he has from the bottom up, coming from a math background.

His work at HireBeat adn KPMG was really intence in datascience, with a strong bias toward Machine learning at hirebeat. 

He has a very strong background in NLP, and even got some experience in reinforcement learning from his project in black jack AI.

He never worked with Time Series before, only used random forest and logistic regression. He called the XGBoost package before but doesn't know its implemation details.

TL;DR: He seemed like he would have a lot of ML expertise, but when we discussed it further, it became clear that he had really deep understanding of some implementations, but really never used any of the tools used in the industry today like BERT for NLP or XGboost for ML. With that in mind he might not be a good match for USAA, and might not be best suited for the internal algo. He has definitely the expertise to work on Prescient though.
